azure-pipelines.yml

```
@@ -243,22 +243,22 @@ jobs:
   - task: Cache@2
     inputs:
       key: 'PYVISTA_DOC_EXAMPLES_CACHE | "$(PYVISTA_VERSION)"'
-      path: '$(Build.SourcesDirectory)/docs/examples'
+      path: '$(Build.SourcesDirectory)/doc/examples'
     displayName: Build pyvista doc examples cache
 
   # run this twice since VTK9 fails to close render windows
   - script: |
-      make -C docs phtml || true  # will fail on VTK9
-      make -C docs phtml -b coverage SPHINXOPTS="-W"
-      cat docs/_build/coverage/python.txt
+      make -C doc html || true  # will fail on VTK9
+      make -C doc html -b coverage SPHINXOPTS="-W"
```

Can't we keep `phtml` for speed?
```suggestion
      make -C doc phtml || true  # will fail on VTK9
      make -C doc phtml -b coverage SPHINXOPTS="-W"
```

---

Sorry, yes, we can.